en/tutorials-and-examples/blog-auth-example/auth.rst

```diff
@@ -301,6 +301,9 @@ logged in user as a reference for the created article::
         if ($this->request->is('post')) {
             // Added this line
             $article->user_id = $this->Auth->user('id');
+            // Also can be used the two lines
+            //$newData = ['user_id' => $this->Auth->user('id')];
+	    //$article = $this->Articles->patchEntity($article, $newData);
```

Comment:
Indentation looks inconsistent and the phrase might be better as 'You could also do the following'